app/actors/hyrax/actors/create_with_files_actor.rb

```diff
@@ -0,0 +1,62 @@
+module Hyrax
+  module Actors
+    # Creates a work and attaches files to the work
+    class CreateWithFilesActor < Hyrax::Actors::AbstractActor
```

Comment:
What change was needed here that required us overriding the (presumably) upstream `CreateWithFilesActor` ?